src/nft/components/collection/Activity.tsx

```diff
@@ -116,17 +125,21 @@ export const Activity = ({ contractAddress, rarityVerified, collectionName, chai
   const Filter = useCallback(
     function ActivityFilter({ eventType }: { eventType: ActivityEventType }) {
       const isActive = activeFilters[eventType]
+      const activeBackgroundColor = isDarkMode ? vars.color.gray500 : vars.color.gray200
 
       return (
-        <Box
-          className={clsx(styles.filter, isActive && styles.activeFilter)}
+        <FilterBox
+          className={clsx(styles.filter)}
+          style={{
+            background: isActive ? activeBackgroundColor : themeVars.colors.backgroundInteractive,
```

Comment:
nit: I think you should match what you did in AssetDetails where the background color was passed into the styled component

Reply:
causes an error

Reply:
fixed